src/main/java/com/fasterxml/jackson/core/io/IOContext.java

```diff
@@ -199,6 +233,16 @@ public StreamWriteConstraints streamWriteConstraints() {
         return _streamWriteConstraints;
     }
 
+    /**
+     * Returns : {@link ErrorReportConfiguration}, container for configuration values used when 
+     * handling errorneous token inputs. 
+     *
+     * @since 2.16
+     */
+    public ErrorReportConfiguration getErrorReportConfiguration() {
```

Comment:
Let's follow the New Jackson Naming convention :)

and drop "get", so `errorReportConfiguration()` (like `streamWriteConstraints()`)

Reply:
Ok, I notice @pjfanning requested `getXxx()` accessor, so it was "right way" at first. We are moving to reduce use of "get" in cases where there is no existing convention (for particular class) -- `get` and `set` method are to be used for general purpose properties/attributes, but not for configuration.
For time being there is obviously some inconsistency: for 3.0 more of get/set prefixes are being/have been removed. But there's no urgency for 2.x.